entrypoints/popup/App.tsx

```
@@ -34,15 +51,18 @@
+          {t("button.count", { count })}
         </Button>
-        <p className="text-muted-foreground">
-          Edit <code className="text-primary">src/App.tsx</code> and save to
-          test HMR
-        </p>
-        <p className="text-muted-foreground">
-          Click on the WXT and React logos to learn more
-        </p>
+        <p
+          className="text-muted-foreground"
+          dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)